src/main/resources/db/changelog/changes/schema.xml

```diff
@@ -333,15 +333,15 @@
         <insert tableName="user">
             <column name="id" valueSequenceNext="user_id_seq"/>
             <column name="role_id" valueSequenceCurrent="user_role_id_seq"/>
-            <column name="user_login" value="rus_admin"/>
+            <column name="user_login" value="rus_admin@northsixty.com"/>
             <column name="user_name" value="Russian Admin"/>
             <column name="user_password" value="$2a$08$sp14HY.XnK2w.qRE7fqMA.ja2aSL5sFGMSPSPdk/nHK.2Lj4GgK.q"/>
             <column name="language" value="rus"/>
         </insert>
         <insert tableName="user">
             <column name="id" valueSequenceNext="user_id_seq"/>
             <column name="role_id" valueSequenceCurrent="user_role_id_seq"/>
-            <column name="user_login" value="eng_admin"/>
+            <column name="user_login" value="eng_admin@northsixty.com"/>
```

Comment:
Do we have an access to the emails?

Reply:
Yep, the service even promises that this temp emails will be active until we delete them manually.